rcon/player_history.py

```diff
@@ -275,10 +294,31 @@ def add_player_to_blacklist(steam_id_64, reason, name=None):
                     steamid=player, is_blacklisted=True, reason=reason)
             )
 
+        try:
+            if rcon and name and ban_if_in_game:
```

Comment:
Depending on the Rcon here makes things a bit intertwined don't you think?

Also it's a big change in behaviour, so if we keep that let's annonce it quite explicitly in the release

Reply:
It's a minor feature so we can strip it out